.github/workflows/tests-l4lb.yaml

```diff
@@ -0,0 +1,69 @@
+name: Cilium L4LB XDP
+
+# Any change in triggers needs to be reflected in the concurrency group.
+on:
+  issue_comment:
+    types:
+      - created
+  # Run every 6 hours
+  schedule:
+    - cron:  '0 4/6 * * *'
+  ### FOR TESTING PURPOSES
+  # pull_request:
+  #  types:
+  #    - "labeled"
+  ###
+
+concurrency:
+  # In case of PR comment, we can't simply append the comment to the group name
+  # as that is too long and results in an error. Instead we check if it's a
+  # trigger phrase and append a simple 'trigger-phrase'.
+  group: |
+    "${{ github.workflow }} ${{ github.event.issue.pull_request.url || 'scheduled' }}
+     ${{ (startsWith(github.event.comment.body, 'ci-l4lb') ||
+          startsWith(github.event.comment.body, 'test-me-please')) && 'trigger-phrase' }}"
```

Comment:
Do we plan on having this test required for merging PRs? If that's the case, I expect that it will become a serious bottleneck for merging PRs, since number of macos runners is very limited.

Can we run this only for changes that touch load balancer (alternatively, we can have a skipping mechanism akin to bpf lint workflow: https://github.com/cilium/cilium/blob/master/.github/workflows/lint-bpf-checks.yaml).

Reply:
> ...since number of macos runners is very limited

Oh, I've not noticed this! To this is even a greater reason to run this elsewhere, if cgroupsv2 is not available anywhere out of the box, an EKS setup with a custom node AMI should be quite doable.

Reply:
Actually, I'd check Bottlerocket, it might have cgroupsv2, you can use it on EKS instead of AL2.

Reply:
I think it's 5 concurrent macos runners: https://docs.github.com/en/actions/reference/usage-limits-billing-and-administration

Reply:
I don't think a limit of 5 concurrent jobs will be an issue. The job currently takes less than 15min. So we would need to run 20 end-to-end tests per hour (19 if you take the scheduled workflow into account) just to reach the limit. We are not yet that busy a project :smile:

Reply:
Didn't check how long were the runs. 15 mins should be fine, but I would definitely keep an open mind about moving this somewhere else.

Reply:
Limits are only one end of the stick. Apple doesn't allow to run macOS on any random VMs on any host machine, it has to run on Apple hardware. The hardware is going to be in much more limited supply and is more costly. The usage limits guard against too high of a demand, but they don't guarantee availability.